src/components/ProjectDomain/projectdomainTable.tsx

```diff
@@ -77,7 +85,9 @@ const ProjectDomainTable: React.FC = () => {
   const handleDelete = async (domainId: string) => {
     try {
       await apiHelperService.deleteProjectdomain(domainId);
-      fetchDomainData(); // Re-fetch data after deletion
+      const updatedDomainDict = { ...domainDataDict };
+      delete updatedDomainDict[domainId]; // Remove the deleted domain
```

Comment:
Don't need to update delete functionality, revert this back to call new data api.
Fetching fresh data is better than maintaining deleted data from state